test/index.test.js

```diff
@@ -28,6 +28,37 @@ test('[useCallback] sinon-style', function(assert) {
   });
 });
 
+test('[useCallback] sinon-style onCall', function(assert) {
+  var getObject = AWS.stub('S3', 'getObject');
+  getObject.onCall(0).yields(null, data);
```

Comment:
Out of curiosity, this failed prior to upgrade? I feel like I've used `.onCall()` in the past 🤔